### PR TITLE
Removed a now unnecessary compatibility fix

### PR DESCRIPTION
**Pull request by freddrichards**
Removed a no longer necessary compatibility fix for deal.II 8.5.

### CODE REVIEW

**Comment by tjhei** [comment]
It looks like your email in the commit is not set correctly. Can you talk to one of us about this?

**Comment by freddrichards** [comment]
I’ve now configured git with my email address. I just submitted a manual-related pull request which John dealt with and I think it worked so hopefully this is fixed now.

Best,

Fred

**Comment by tjhei** [comment]
No, you will need to rewrite the history for this PR and update it (you can tell that the commit above does not have your github icon)

File: source/simulator/melt.cc [hunk 1]
```diff
@@ -578,11 +578,8 @@ namespace aspect
         face->boundary_id(),
         scratch.face_material_model_inputs,
         scratch.face_material_model_outputs,
-#if DEAL_II_VERSION_GTE(9,0,0)
         scratch.face_finite_element_values.get_normal_vectors(),
-#else
-        scratch.face_finite_element_values.get_all_normal_vectors(),
-#endif
+
```
Comment (tjhei) [inline source/simulator/melt.cc]: please remove this empty line

**Pull request merged**